src/GameModel.h

```diff
@@ -657,6 +657,8 @@ namespace gameModel{
          * @return
          */
         auto clone() const -> std::shared_ptr<Environment>;
+
+        auto getFreeCellsForRedeploy(const gameModel::TeamSide &teamSide, const std::shared_ptr<gameModel::Environment> &env) -> const std::vector<gameModel::Position>;
```

Comment:
ein Doxygen Kommentar wäre schön ;)

Reply:
Warum lassen alle immer die erste Zeile in den Kommentaren frei? Die hat schon ihre Daseinsberechnung ;)